ts/src/sdl/SDL/SDL.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
@stalniy I think this is the most important file to review, since it is related to the core implementation.